vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,104 @@
+def call(Map pipelineParams) {
+    pipeline {
+        agent any
+        parameters {
+            string(name: 'scylla_version', defaultValue: '', description: 'Scylla version to test')
+            string(name: 'base_versions', defaultValue: '', description: 'Base versions')
+            string(name: 'provision_type', defaultValue: '', description: 'Provision type')
+            string(name: 'new_scylla_repo', defaultValue: '', description: 'New Scylla repo')
+            string(name: 'matrix_storage', defaultValue: '', description: 'This parameter is used for trigger with Scylla master version only. It point which kind of test to run: vnodes or tablets. Expected values: master-tablets OR master-vnodes')
+        }
+        stages {
+            stage('Get Scylla Version') {
+                steps {
+                    script {
+                        def scylla_version = params.scylla_version?.trim()
+                        def matrix_storage = params.matrix_storage?.trim()
+                        println("Scylla version: ${scylla_version}")
+                        println("Matrix storage: $matrix_storage")
+                        if (scylla_version == "master:latest") {
+                            scylla_version = "master"
+                            if (!matrix_storage) {
+                                error "Test type (vnodes or tablets) is not set. Please provide one of a valid 'matrix_storage' values: 'master-tablets' OR 'master-vnodes'"
+                            }
+                        }
+                        triggers {
```

Comment:
I don't think this is the correct place for trriggers...

it should be in the same level as parameters, under pipeline.

It might not worked places here